packages/next/src/template-renderer/index.tsx

```diff
@@ -54,6 +55,9 @@ export async function TemplateRenderer( {
 			</TemplateScripts>
 			{ /* Hot Fix for adding classes to the body outside the root layout */ }
 
+			{ /* @todo: Check if we can patch this with either the portal or the @wordpress/hooks implementation. */ }
+			<LightboxOverlay />
```

Comment:
Put this (todo included) in `<TemplateFooter>`, that way it's just the footer "portal" that will need to change and not our `TemplateRenderer`.

Reply:
We don't have a `<TemplateFooter>`. Do you mean putting this inside `<TemplateScripts>`?

Reply:
No I meant to create a new wrapper component that matches the DX and naming conventions of the existing component.

That component will hold our manual injections like this lightbox until we can turn it into a portal